Telran_HomeWork/src/telran/Pizzeria20200413/PizzeriaTest.java

```diff
@@ -0,0 +1,38 @@
+package telran.Pizzeria20200413;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class PizzeriaTest {
+    PizzaInterface p = new PizzaBaseImpl();
+    @Test
+    public void testPizzeria() {
+
+        double actual = p.getPrice();
+        double expekted = 3.0;
+        assertEquals(expekted, actual, 0.0);
+    }
+    @Test
+    public void testPizzeria1() {
+        PizzaInterface pt = new TomatDecorator(p);
+        double actual = pt.getPrice();
+        double expekted = 4.0;
+        assertEquals(expekted, actual, 0.0);
```

Comment:
Does it actually work? assertEquals(String message, int expected, int actual) -> your actual is 0.0, which is probably wrong. You call assertEquals with wrong parameters.

Reply:
without "delta" in Java8 error - java.lang.AssertionError: Use assertEquals(expected, actual, delta) to compare floating-point numbers. With "delta = 0.0" - all worked. I was surprised too ))

Reply:
Ok, this is delta for double values, should be ok. It is bad idea in general to store price as double because the price has no floating point. It is an integer if you measure in cents.